### DISCUSSION TITLE
AI幫你讀論文的Explainpaper

### DISCUSSION THREAD

**Discussion by kaichiiiiiii** [1101-GS3073-C/Python-Community-GS3073-C#647] [General]
https://www.youtube.com/watch?v=BHT68uqGAUQ   👈示範影片
https://36kr.com/p/2024045824191496   👈別人的評測

這個AI能夠幫助你更快的理解paper的內容，說明段落概要，而且還可以問他問題。
我認為對在閱讀論文時幫助很大，要看的論文越來越多，如何在艱澀的專業術語中提出精簡概要能夠幫助我們更快吸收更多知識。
不過目前似乎還沒有中文版

**Comment by Valentinaa0214** [comment]
之前在Youtube Shorts上有看過該AI的介紹，這種技術真的幫我們省了很多時間，不然每次找參考資料的時候都要花好多時間在看相關的論文資料上，然後還看不出個所以然來...

**Comment by jerry0108566** [comment]
之前在寫論文心得的時候就有用過類似的工具了，真的很方便，節省很多時間

**Comment by HsuChen88** [comment]
這個工具比3秒看完一本書的黑人還有用，不會看完一問三不知。
